### PR TITLE
FF experimental to rewrite

### PR DESCRIPTION
**Pull request by Dougley**
Rewrite is stable (enough) to be considered experimental code.

Things still missing:
- [ ] Docker support
- [ ] Bezerk support
- [ ] Music streaming
- [ ] Most of the commands from v4

### CODE REVIEW

**Comment by Dougley** [comment]
Can't be merged automatically because of the many conflicts, don't bother fixing those.  
Think of this PR more of a discussion.

**Comment by caf203** [comment]
Perhaps we should document what people need to set for environment variables rather than let them through code? I get that this is brand new, but people actually use experimental.

**Comment by Dougley** [comment]
Would a changelog paragraph suffice? I'm writing that right now.

**Comment by caf203** [comment]
As long as they're included, that should work until LW writes new docs.

**Comment by linuswillner** [comment]
@zaza7 promised to send me an exhaustive change list, so as soon as I get that I’ll start on the new docs.

**Comment by Dougley** [comment]
These are all the environment variables we currently use, these are also in the changelog i have staged locally. 

| Variable | Required | Function |
|----------|----------|----------|
| `BOT_TOKEN` | **Mandatory** | The token to use |
| `BOT_PREFIX` | **Mandatory** | The prefix to use |
| `WILDBEAST_MASTERS` | **Mandatory** | String of user IDs separated by pipe characters that should be considered master users |
| `WILDBEAST_SUPPRESS_COMMANDLOG` | Optional | If set, suppresses loglevel `CMD` |
| `WILDBEAST_PREFERRED_DATABASE` | Optional | Override the database driver used, defaults to `arangodb` |
| `ELASTICSEARCH_URI` | Optional | If set, configure logging to log to Elasticsearch |
| `ELASTICSEARCH_INDEX` | Optional | Override default index of `wildbeast` for Elasticsearch logging |
| `SENTRY_DSN` | Optional | If set, configure error logging to report to Sentry as well |
| `ARANGO_URI` | Optional | If set, override the default Arango URI of `http://localhost:8529` |
| `ARANGO_USERNAME` | Required if running default | Set the username used for authenticating with Arango |
| `ARANGO_PASSWORD` | Required if running default | Set the password used for authenticating with Arango |
| `ARANGO_DATABASE` | Optional | If set, override the default database used, defaults to `wildbeast` |
| `NODE_ENV` | Optional | If set to `debug`, enable verbose logging |

File: README.md [hunk 1]
```diff
@@ -14,21 +11,14 @@
 
 ---
 
-WildBeast is a multifunction Discord bot intended to be able to perform various tasks, ranging from easy server moderation (Kick, ban etc.) to having fun (Cleverbot etc.) in your Discord server. Written in JavaScript [Node.JS](https://nodejs.org), using the [discordie](https://qeled.github.io/discordie) library.
+WildBeast is a multifunctional Discord bot, intended to provide a framework thats both easy to use and run, and easy to extend and modify.
```
Comment (linuswillner) [inline README.md]: Any chance you could word this a little better? A lot of "and" stacking there.

Comment (Dougley) [reply README.md]: Changed locally to `WildBeast is a multifunctional Discord bot, intended to provide a framework that's easy to use, extend, and modify.`

**Comment by Dougley** [comment]
The CI config is migrated locally.

**Pull request merged**